packages/styles/scss/components/masthead/_masthead-search.scss

```diff
@@ -77,20 +77,6 @@
       }
     }
 
-    .#{$prefix}--header__nav {
-      &::before {
-        content: '';
-        display: block;
-        position: absolute;
-        right: 0;
-        top: 50%;
-        transform: translateY(-50%);
-        height: 1.5rem;
-        width: 0.0625rem;
-        background-color: #dcdcdc;
-      }
-    }
```

Comment:
Thanks for this @proeung! This removes the 1px horizontal rule between the logo and the platform name. Is that intended? Because I don't see it in the design specs.

![image](https://user-images.githubusercontent.com/909118/117142055-454b1d80-ad7d-11eb-9127-ce7ccbbf8f9c.png)

Reply:
See my comment here - https://github.com/carbon-design-system/carbon-for-ibm-dotcom/pull/5922#issuecomment-837454977